.changes/config.json

```diff
@@ -240,10 +246,6 @@
         {
           "path": "${ pkg.path }/target/package/tauri-cli-${ pkgFile.version }.crate",
           "name": "${ pkg.pkg }-${ pkgFile.version }.crate"
-        },
-        {
-          "path": "${ pkg.path }/schema.json",
-          "name": "schema.json"
```

Comment:
why removing it from the cli release?

Reply:
since the schema is generated from tauri-build, it makes sense to upload it to tauri-build release.  In the vscode extension, I will check for tauri-build version and download the schema from the release.

Reply:
let's see if no one complains :D at least the schema is still inside the CLI crate itself